src/Controller/ContentElement/ColumnsStartController.php

```diff
@@ -87,7 +88,7 @@ public function generate()
 	 * @param  array $data Data array
 	 * @return array       Columns configuration
 	 */
-	public static function getColumnsConfiguration(array $data)
+	public static function getColumnsConfiguration(array $data): array
```

Comment:
Can this method be `private`?

Reply:
No, this method is reused by the custom elements extension: https://github.com/madeyourday/contao-rocksolid-custom-elements/blob/4b029c297a0f635d728ad048d3e1741dd95b1256/src/Element/CustomElement.php#L274